tests/unit_tests/config/test_queue_config.py

```diff
@@ -67,18 +67,23 @@ def test_project_code_is_not_overwritten_if_set_in_config(queue_system):
 
 
 @pytest.mark.usefixtures("use_tmpdir", "set_site_config")
-@given(st.integers(min_value=1, max_value=300))
-def test_that_an_invalid_queue_system_provided_raises_validation_error(num_real):
+@pytest.mark.parametrize("invalid_queue_system", ["VOID", "BLABLA", "GENERIC", "*"])
+def test_that_an_invalid_queue_system_provided_raises_validation_error(
+    invalid_queue_system,
+):
+    """There is actually a "queue-system" called GENERIC, but it is
+    only there for making queue options global, it should not be
+    possible to try to use it"""
     filename = "config.ert"
 
     with open(filename, "w", encoding="utf-8") as f:
-        f.write(f"NUM_REALIZATIONS {num_real}\nQUEUE_SYSTEM VOID\n")
```

Comment:
num_real is taken out of the test, as it is pointless. Split out into a separate commit or ok?

Reply:
split.